src/panoptes/pocs/tests/test_pocs.py

```diff
@@ -179,31 +202,6 @@ def test_is_weather_safe_no_simulator(pocs):
     assert pocs.is_weather_safe() is False
 
 
-def test_unsafe_park(pocs):
-    pocs.set_config('simulator', 'all')
-    pocs.initialize()
-    assert pocs.is_initialized is True
-    os.environ['POCSTIME'] = '2020-01-01 08:00:00'
-    assert pocs.state == 'sleeping'
-    pocs.get_ready()
-    assert pocs.state == 'ready'
-    pocs.schedule()
-    assert pocs.state == 'scheduling'
-
-    # My time goes fast...
-    os.environ['POCSTIME'] = '2020-01-01 18:00:00'
-    pocs.set_config('simulator', hardware.get_all_names(without=['night']))
-
-    assert pocs.is_safe() is False
```

Comment:
Why is this test removed? Could we change this call using the new `ignore` param and make test work?

Reply:
I think this test is pretty much broken, but I've replaced it with another `test_unsafe_park` which tests the same functionality.